policy/modules/system/authlogin.if

```diff
@@ -938,6 +938,24 @@ interface(`auth_setattr_faillog_files',`
 	setattr_files_pattern($1, faillog_t, faillog_t)
 ')
 
+########################################
+## <summary>
+##	Search faillock dirs (/run/faillock).
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`auth_search_faillog_dirs',`
```

Comment:
Please remove the `_dirs` from the name, since `search` implies dirs already. Also move the interface above `auth_append_faillog` (line 847)